aim/web/ui/src/components/ImagePropertiesPopover/ImagePropertiesPopover.tsx

```diff
@@ -57,7 +57,7 @@ IImagePropertiesPopoverProps): React.FunctionComponentElement<React.ReactNode> {
             Image Size:
           </Text>
           <Text className='ImagePropertiesPopover__sizePercent'>
-            {imageProperties.imageSize}%
+            {imageProperties.mediaItemSize}%
```

Comment:
mediaProperties

Reply:
I think that component is working with images explore only.